reflex/state.py

```diff
@@ -1405,6 +1419,24 @@ def on_load_internal(self) -> list[Event | EventSpec] | None:
             type(self).set_is_hydrated(True),  # type: ignore
         ]
 
+    def __getstate__(self):
```

Comment:
This leaves `router_data` on each substate. Is that the intended behavior?

Reply:
For now, yes. But I might be able to get away with stripping it out.

Reply:
Maybe `router_data` turns out to be a substate on its own in the end :smile:

Reply:
@abulvenz not a bad idea actually.. it may involve having to save two states each time as a downside, but it might be a cleaner separation in the future.